repos/c2sm/packages/py-icon4py/package.py

```diff
@@ -25,10 +25,12 @@ class PyIcon4py(PythonPackage):
 
     version('main', branch='main', git=git)
     version('0.0.3', tag='v0.0.3', git=git)
+    version('0.0.3.1', tag='v0.0.3.1', git=git)
     version('0.0.4', tag='v0.0.4', git=git)
     version('0.0.5', tag='v0.0.5', git=git)
     version('0.0.6', tag='v0.0.6', git=git)
     version('0.0.7', tag='v0.0.7', git=git)
+    version('0.0.8', tag='v0.0.8', git=git)
```

Comment:
Note that this version has been remove by @halungge because it needs all deps of the green-line (or blue line?) too.

Reply:
Ah okay. Thanks! I'll remove it.